internal/show/versions.go

```diff
@@ -67,19 +94,119 @@ func printDeploymentVersions(clusterInfo *cluster.Info, printer *table.Printer,
 		}
 
 		version, repository := images.ParseOperatorImage(deployment.Spec.Template.Spec.Containers[0].Image)
-		printer.Add(component, repository, version)
+
+		runningVersion, err := getVersionForComponent(clusterInfo, component,
+			labels.SelectorFromSet(deployment.Spec.Selector.MatchLabels).String())
+		if err != nil {
```

Comment:
Should this be considered an error, or just return version as `Unavailable`? Error handling was useful while working on this change, but throwing an error means `show versions` output can be incomplete or even empty if just one component ran into error.

Reply:
Right, it shouldn’t be an error.